### DISCUSSION TITLE
How to do crash triage on c program with image?

### DISCUSSION THREAD

**Discussion by pollyMath** [AFLplusplus/AFLplusplus#1620] [Q&A]
Hello! I am trying run fuzzer on a simple c program that takes png image as input. I have managed to run the fuzzer but I am stuck how to do the crash triage. As far as I understood, I need to use a debugger and run sth like:

`gdb -q ./test afl_out/default/queue/id:000002,src:000001,time:533,execs:146,op:havoc,rep:16,+cov`

however, I get /afl_out/default/queue/id:000002,src:000001,time:533,execs:146,op:havoc,rep:16,+cov is not a core dump: file format not recognized

Now I am unsure is that the crash or if I am running something wrong and I cannot find much info online.
Help on how to do this would be greatly appreciated!

**Comment by vanhauser-thc** [comment]
you should learn how to use gdb. this is the wrong place.
```
gdb ./test
   run afl_out/default/queue/id:000002,src:000001,time:533,execs:146,op:havoc,rep:16,+cov
```
